src/bootstrap.coffee

```diff
@@ -169,7 +169,7 @@ bootstrapOrRetry = ->
 
 hasDeviceApiKeySupport = (osVersion) ->
 	try
-		!/^Resin OS /.test(osVersion) or semver.gte(semverRegex.test(osVersion), '2.0.2')
+		!/^Resin OS /.test(osVersion) or semver.gte(semverRegex.exec(osVersion)[0], '2.0.2')
```

Comment:
This looks fine but might be better to let semver parse the version, i.e. with something like:

```
version = osVersion.replace(/^Resin OS /, '')
return false if version is osVersion # not Resin OS
return semver.valid(version) and semver.gte(version, '2.0.2')
```